templates/repo/actions/runs_list.tmpl

```diff
@@ -12,7 +12,7 @@
 			</div>
 			<div class="flex-item-main">
 				<a class="flex-item-title" title="{{.Title}}" href="{{if .Link}}{{.Link}}{{else}}{{$.Link}}/{{.Index}}{{end}}">
-					{{- .Title -}}
+					{{if .Title}}{{- .Title -}}{{else}}&lt;{{ctx.Locale.Tr "actions.runs.empty_commit_message"}}&gt;{{end}}
```

Comment:
I think it's better to get rid of such `&lt;` in code.

Even `runs.empty_commit_message = (empty commit message)` seems better.

Reply:
Done.